pico-statsd/src/main/scala/org/pico/statsd/syntax/event/package.scala

```diff
@@ -11,15 +11,15 @@ package object event {
   
   //-------------------- METRIC --------------------------------------------
   implicit class SinkSourceOps_Metric_Rht98nT[A, B](val self: SinkSource[A, B]) extends AnyVal {
-    def withMetrics(aspect: String, sampleRate: Option[Double], extraTags: String*)(implicit c: StatsDClient, m: Metric[B]): SinkSource[A, B] = {
-      self += self.effect(sendMetrics(c, aspect, sampleRate.getOrElse(1d), extraTags.toList, m))
+    def withMetrics(aspect: String, sampleRate: Option[Double], tags: String*)(implicit c: StatsDClient, m: Metric[B]): SinkSource[A, B] = {
```

Comment:
`extraTags` has been renamed to `tags` so that it is consistent throughout the whole file.